htdocs/core/lib/date.lib.php

```diff
@@ -130,7 +130,7 @@ function dol_time_plus_duree($time, $duration_value, $duration_unit, $ruleforend
 	if ($duration_unit == 's') {
 		return $time + (int) ($duration_value);
 	}
-	if ($duration_unit == 'i' || $duration_unit == 'mn' || $duration_unit == 'min') {
+	if ($duration_unit == 'i' || $duration_unit == 'min') {
```

Comment:
For compatibility with external modules, we should keep the compatiblity test
$duration_unit == 'mn'
too

Reply:
There is update in 20.0.0-21.0.0.sql that change the code, the code "mn" is from 20.0